src/connectors/autocomplete/__tests__/connectAutocomplete-test.ts

```diff
@@ -566,4 +566,155 @@ search.addWidgets([
       );
     });
   });
+
+  describe('sendEvent', () => {
+    let firstIndexHits;
+    let secondIndexHits;
+    let instantSearchInstance;
+    let render;
+    beforeEach(() => {
```

Comment:
this style is ok, but in other files / tests we have more used function like `createInitialisedWidget`, you could have a  similar `createRenderedWidget`?

Reply:
https://github.com/algolia/instantsearch.js/pull/4464/commits/669501559bff1fda2a340c02658b87fe574053a9